archive-index.html

```diff
@@ -92,13 +92,80 @@ <h1 class="page-header">
                   of things unknown <br>
                   but longed for still”
                 </p>
-                <a class="btn btn-primary" href="#">Read More <span class="glyphicon glyphicon-chevron-right"></span></a>
-
+<!--  Modal -->              
+                <button class="btn btn-primary" data-toggle="modal" data-target="#poem1">Read More <span class="glyphicon glyphicon-chevron-right"></span></button>
+
+                <div class="modal fade" id="poem1" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
Kira, I think it's great what you're doing with the modals in the archive. Nice job!